### PR TITLE
feat: menambahkan layout login

### PR DESCRIPTION
**Pull request by RutriAyut**
Feat : Login Page

- [x] Form login
- [x] Forgot password button
- [x] Sign in button
- [x] Sign in with google button
- [x] Responsive
- [ ] Redirect after login

Nb : Redirect after login belum bisa tapi untuk input email dan password sudah bisa di console log

1440
image

768
image

320
image

### CODE REVIEW

File: src/pages/login/components/LoginForm.tsx [hunk 1]
```diff
@@ -2,85 +2,85 @@ import { Button } from "@/components/ui/button";
...
-          Github
-        </Button>
-        <Button variant="outline" type="button" disabled={isPending}>
+      <div className="grid">
+        <Button variant="outline" type="button" disabled={isPending} className="rounded-xl border-gray-200 gap-2">
           {isPending ? (
             <Loader className="mr-2 h-4 w-4 animate-spin" />
           ) : (
-            <Facebook className="mr-2 h-4 w-4" />
+            <img src="./src/assets/google.svg" />
```
Comment (bayuuat) [inline src/pages/login/components/LoginForm.tsx]: buat Image dibuat kaya mas Saiful gini ya mba, biar ke load di website. Soalnya kalo pake src ga kebaca.

https://github.com/SYNRGY-6-TEAM-5/frontend-web/blob/840c99f25837fb56d00bd105387569616eb0d81f/src/components/containers/HeroSection.tsx#L41

image

**Review by bayuuat — CHANGES_REQUESTED** [review]

File: src/pages/login/components/LoginForm.tsx (hunk 1 above)
Comment (Mwazowsky) [reply src/pages/login/components/LoginForm.tsx]: Atau mungkin biar ga banyak banyak asset static nya untuk icon simple kyk google, show password bisa pakai [phosporus-icon](https://phosphoricons.com/)

Makainya gini

```suggestion
import { GoogleLogo  } from "@phosphor-icons/react";

<GoogleLogo size={24} weight="bold" />
```

**Review by Mwazowsky — COMMENTED** [review]
Keep up the great work

**Review by Mwazowsky — APPROVED** [review]
Okay

**Review by bayuuat — APPROVED** [review]

**Pull request merged**